doc/ca65.sgml

```diff
@@ -854,8 +899,6 @@ cheap local labels are written to the object file and will be available in the
 symbol file via the linker. Unnamed labels are not written to the object file,
 because they don't have a name which would allow to access them.
 
-
-
 <sect>Scopes<label id="scopes"><p>
```

Comment:
Look at <https://github.com/cc65/wiki/wiki/LinuxDoc-sources-style-guide> for a guide about the line spacing between sections.